src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook_test.go

```diff
@@ -279,6 +279,38 @@ func TestValidateUpdate_NoError(t *testing.T) {
 
 		assert.Error(t, err)
 	})
+
+	t.Run("proxy subdomain must be the same as kafka subdomain", func(t *testing.T) {
```

Comment:
I think you need to add the tests to both Create and Update.

Reply:
Thanks, added.